code/game/objects/items/weapons/tools.dm

```diff
@@ -199,6 +256,32 @@
 	icon_state = "cutters_cyborg"
 	toolspeed = 2
 
+/obj/item/weapon/wirecutters/power
+	name = "Jaws of Life"
+	desc = "A set of jaws of life, the magic of science has managed to fit it down into a device small enough to fit in a tool belt. It's fitted with a cutting head"
+	icon_state = "jaws_cutter"
+	item_state = "jawsoflife"
+	origin_tech = "materials=2;engineering=6"
+	materials = list(MAT_METAL=150,MAT_SILVER=50,MAT_TITANIUM=25)
+	usesound = 'sound/items/jaws_cut.ogg'
+	toolspeed = 4
+
+/obj/item/weapon/wirecutters/power/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is the [src.name] around \his neck, it looks like \he's trying to rip \his head off!</span>")
+	playsound(loc, 'sound/items/jaws_cut.ogg', 50, 1, -1)
+	var/mob/living/carbon/C = user

```

Comment:
I forgot about maintenance drones, but aren't the only dextrous animals monkeys? and aren't they also a carbon mob?